Opinion/slowroads-handling-editor
# Add greasemonkey support

## Tested userscript extensions (Firefox, latest update):
 - Tampermonkey, confirmed working
 - Violentmonkey, confirmed working
 - Greasemonkey, confirmed working

## Test procedure (for each extension):
 1. Disable userscript from other extensions
 2. Enable userscript on the extension being tested
 3. Reload the game page and begin the game
 4. Edit 'accel' value and see if acceleration changed
 5. Press the 'reset handling' button and see if acceleration changed back
